python/lsst/sattle/sattlePy.py

```diff
@@ -337,9 +365,36 @@ def _extend_line(self, x1, y1, x2, y2, extend_length):
         unit_y = dy / length
 
         # Extend the line on both ends by the length
-        x1_extended = x1 - extend_length * unit_x
-        y1_extended = y1 - extend_length * unit_y
-        x2_extended = x2 + extend_length * unit_x
-        y2_extended = y2 + extend_length * unit_y
-
-        return x1_extended, y1_extended, x2_extended, y2_extended
+        x1 = x1 - extend_length * unit_x
+        y1 = y1 - extend_length * unit_y
+        x2 = x2 + extend_length * unit_x
+        y2 = y2 + extend_length * unit_y
+
+        return x1, y1, x2, y2
+
+    def _check_corners(self, corner):
+        """ After all the calculations, make sure none of the coordinates
+        exceed the spherical coordinates. If they do correct them."""
+        lon, lat = corner[0], corner[1]
+
+        # Check lat first in case we flip over the poles
+        # Don't really need this one but whatever
+        # these names are terrible
```

Comment:
Consider making the names less terrible 😄

Reply:
Moved work from another ticket to improve this section